hawc/apps/lit/api.py

```diff
@@ -62,9 +62,11 @@ def reference_tags(self, request, pk):
         df = models.ReferenceTags.objects.as_dataframe(instance.id)
         return Response(df)
 
+
```

Comment:
can you revert the changes in this file? just delete the extra lines which were added.

Reply:
This goes with above. I know Danny and I opened these files and did something a yesterday when I was having trouble accessing the API urls, and I was unable to automatically merge the master branch with mine. I don't really understand it though, should it just match what's in the master branch for this file?

Reply:
just delete the blank lines that were added here, so that when you push the changes up the file disappears from this "diff" view

Reply:
Also, I just realized I don't know how to build an r package - can you add some instructions for how to build the package in a new section in `docs/development.rst`?